### PR TITLE
[OSDOCS#9544]: Release Notes 4.13.32

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-9544

Link to docs preview:
https://71260--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-32

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
LGTM!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -3725,3 +3725,33 @@ $ oc adm release info 4.13.31 --pullspecs
...
+
+[id="ocp-4-14-11-features"]
+==== Features
+
+The following feature is included in this z-stream release:
+
+[id="ocp-4-13-32-wherabouts-cron-schedule"]
+===== Enabling configuration of whereabouts cron schedule
+
+* Previously, the Wherabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts' cron schedule. (link:https://issues.redhat.com/browse/OCPBUGS-27958[*OCPBUGS-27958*])
```
Comment (cbippley) [inline release_notes/ocp-4-13-release-notes.adoc:3753]:
```suggestion
* The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts cron schedule. xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
```

Comment (obrown1205) [reply release_notes/ocp-4-13-release-notes.adoc]: done :) ty!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -3725,3 +3725,33 @@ $ oc adm release info 4.13.31 --pullspecs
...
+$ oc adm release info 4.13.32 --pullspecs
+----
+
+[id="ocp-4-14-11-features"]
+==== Features
+
+The following feature is included in this z-stream release:
+
+[id="ocp-4-13-32-wherabouts-cron-schedule"]
+===== Enabling configuration of whereabouts cron schedule
```
Comment (adellape) [inline release_notes/ocp-4-13-release-notes.adoc:3751]:
```suggestion
===== Enabling configuration of Whereabouts cron schedule
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -3725,3 +3725,33 @@ $ oc adm release info 4.13.31 --pullspecs
...
+
+* The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts cron schedule. xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
```
Comment (adellape) [inline release_notes/ocp-4-13-release-notes.adoc:3753]: - "Hardcoded" one word per IBM SG
- Markup `ConfigMap` and add "object" or "resource" after
- Initcap "Whereabouts"
- Feels like the xref is missing a "For more information" or something to make it a complete sentence?

```suggestion
* The Whereabouts reconciliation schedule was hardcoded to run once per day and could not be reconfigured. With this release, a `ConfigMap` object has enabled the configuration of the Whereabouts cron schedule. For more information, see xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
```

**Review by adellape — APPROVED** [review]

**Pull request merged**